iroha_client_no_std/src/tx.rs

```diff
@@ -0,0 +1,161 @@
+//! This module contains Transaction related functionality of the Iroha.
+//!
+//! `RequestedTransaction` is the start of the Transaction lifecycle.
+
+use crate::prelude::*;
+use iroha::crypto::{Hash, KeyPair, Signature};
+use iroha_derive::Io;
+use parity_scale_codec::{Decode, Encode};
+use std::time::SystemTime;
+
+/// This structure represents transaction in non-trusted form.
+///
+/// `Iroha` and its' clients use `RequestedTransaction` to send transactions via network.
+/// Direct usage in business logic is strongly prohibited. Before any interactions
+/// `accept`.
+#[derive(Clone, Debug, Io, Encode, Decode)]
+pub struct RequestedTransaction {
+    payload: Payload,
+    signatures: Vec<Signature>,
+}
+
+#[derive(Clone, Debug, Io, Encode, Decode)]
+struct Payload {
+    /// Account ID of transaction creator.
+    account_id: <Account as Identifiable>::Id,
+    /// An ordered set of instructions.
+    instructions: Vec<Instruction>,
+    /// Time of creation (unix time, in milliseconds).
+    creation_time: String,
+}
```

Comment:
In the latest commits tx also have ttl in payload. You may have serialization / deserialization problems if it is not present here.

Reply:
@eadventurous thank you.